src/components/portal/__spec__.js

```diff
@@ -17,12 +17,7 @@ describe("Portal", () => {
     beforeEach(() => {
       wrapper = mount(
         <Portal>
-          <Icon
-            tooltipMessage="Test"
-            tooltipAlign="left"
-            tooltipPosition="top"
-            type="tick"
-          />
+          <Icon tooltipMessage="Test" tooltipPosition="top" type="tick" />
```

Comment:
I think we need to document the `tooltipMessage` and `tooltipPosition` props on `Icon`

Reply:
Actually it won't have these props will it, shouldn't this be `Icon` wrapped in `Tooltip`?

Reply:
Yeah agreed I've refactored it to render Tooltip within Icon now